suite-native/message-system/src/components/MessageScreen.tsx

```diff
@@ -0,0 +1,110 @@
+import { Linking } from 'react-native';
+
+import { IconName } from '@suite-common/icons/src';
+import { Box, Button, Pictogram, PictogramVariant, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Message, Variant } from '@suite-common/suite-types';
+
+type MessageScreenProps = {
+    message: Message;
+    defaultTitle?: string;
+    defaultContent?: string;
+    secondaryButtonLabel?: string;
+    onSecondaryButtonPress?: () => void;
+};
+
+const screenStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundSurfaceElevation0,
+    position: 'absolute',
+    justifyContent: 'center',
+    alignItems: 'center',
+    left: 0,
+    right: 0,
+    bottom: 0,
+    top: 0,
+    paddingTop: utils.spacings.extraLarge,
+    paddingHorizontal: utils.spacings.large,
+    marginBottom: utils.spacings.extraLarge,
+    flex: 1,
+}));
+
+const buttonsWrapperStyle = prepareNativeStyle(utils => ({
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    bottom: 0,
+    paddingHorizontal: utils.spacings.large,
+    marginBottom: utils.spacings.extraLarge,
+}));
+
+const variantMap = {
+    info: 'green',
+    warning: 'yellow',
+    critical: 'red',
+} as const satisfies Record<Variant, PictogramVariant>;
+
+const iconVariantMap = {
+    info: 'info',
+    warning: 'warningTriangleLight',
+    critical: 'warningTriangleLight',
+} as const satisfies Record<Variant, IconName>;
+
+export const MessageScreen = ({
+    message,
+    defaultTitle,
+    defaultContent,
+    secondaryButtonLabel,
+    onSecondaryButtonPress,
+}: MessageScreenProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const { variant, headline, content, cta } = message;
+
+    // TODO: We use only English locale in suite-native so far. When the localization to other
+    // languages is implemented, the language selection logic has to be added here.
+    const messageTitle = headline?.en;
+    const messageContent = content.en;
+    const ctaLabel = cta?.label.en;
+    const ctaLink = cta?.link;
+    const isExternalCta = cta?.action === 'external-link';
+
+    const handleCtaPress = () => {
+        if (isExternalCta && ctaLink) {
+            Linking.openURL(ctaLink);
+        } else {
+            // TODO: handle internal link once we introduce them
```

Comment:
Please remove these TODOs. IMO this is just bad to pollute codebase with TODOs. 

No one is ever going to go to that file, notice this TODO and implement it because it will be out of scope for the issue at hand. From product, this is hidden. Remove the comment and create issue, if its needed, but I would argue this is just lazy way (no offense - we all do it 😇) of not doing this stuff. Codebase is full of redundant todos from 5 years ago. 

@PeKne @Nodonisko What's your opinion on this?

Reply:
Let's discuss it after standup.